packages/email-components/src/components/RatingItem/_theme.scss

```diff
@@ -0,0 +1,30 @@
+@use 'sass:map';
+@use '@tokens' as tokens;
+@use '../../scss/settings/dictionaries';
```

Comment:
Is the `dictionaries` used anywhere here?

Reply:
Yes, in Alert component.

Reply:
I mean here. The `@use` is useless here (same as the use of `sass:map`) - you're not using it anywhere.

Please remove it.

Reply:
sorry, my misunderstanding 😄 🤦‍♂️

Reply:
This one is still not resolved. Please remove the first line.